cmd/cluster-controller/main.go

```diff
@@ -12,13 +16,17 @@ import (
 const numThreads = 2
 
 var (
-	kubeconfigPath = flag.String("kubeconfig", "", "Path to kubeconfig")
-	syncerImage    = flag.String("syncer_image", "", "Syncer image to install on clusters")
-	pullMode       = flag.Bool("pull_mode", true, "Deploy the syncer in registered physical clusters in POD, and have it sync resources from KCP")
-	pushMode       = flag.Bool("push_mode", false, "If true, run syncer for each cluster from inside cluster controller")
+	kubeconfigPath  = flag.String("kubeconfig", "", "Path to kubeconfig")
+	syncerImage     = flag.String("syncer_image", "", "Syncer image to install on clusters")
+	pullMode        = flag.Bool("pull_mode", true, "Deploy the syncer in registered physical clusters in POD, and have it sync resources from KCP")
+	pushMode        = flag.Bool("push_mode", false, "If true, run syncer for each cluster from inside cluster controller")
+	autoPublishApis = flag.Bool("auto_publish_apis", false, "If true, the APIs imported from physical clusters will be published automatically as CRDs")
```

Comment:
Go style says it should be `autoPublishAPIs`: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Sure. I'll fix it.